runtime/src/bank_forks.rs

```diff
@@ -15,6 +15,17 @@ use std::{
     time::Instant,
 };
 
+struct SetRootTimings {
+    root: i64,
```

Comment:
doesn't seem like we need this, `root` is available from the caller function

Reply:
removed!